ci/jobs/reboot-worker-management.yml.j2

```diff
@@ -7,4 +7,12 @@ jobs:
     config:
       params:
         AWS_ACC: "{{accounts['management']}}"
+  - .: (( inject meta.plan.fetch-credentials ))
+    config:
+      params:
+        USERNAME_KEY: "/ci/0df691ef-5c06-e638-ae85-d14bb1e0703f-user"
+        PASSWORD_KEY: "/ci/0df691ef-5c06-e638-ae85-d14bb1e0703f-password"
```

Comment:
We should never hardcode parameters in open-source repositories, not even as a temporary measure.